src/controllers/puppeteer-controller.js

```diff
@@ -0,0 +1,60 @@
+export class PuppeteerController {
+    constructor({ puppeteerService }) {
+        this.service = puppeteerService;
+    }
+
+    async scrapeText(req, res) {
+        try {
```

Comment:
Is there a reason why you didn't pull the whole 
   ` try {
            //funciotn()
            );
            if (results.error) {
                res.status(400).send(results);
            } else {
                res.send(results);
            }
        } catch (err) {
            console.error(err);
            res.sendStatus(500);
        }
    }`
into a private function?

Reply:
Mostly just for flexibility, like if the service returns something in a different format.